package.json

```diff
@@ -1,13 +1,28 @@
 {
   "name": "check-node-version",
   "version": "3.3.0",
+  "engines": {
+    "node": ">=8.3.0"
+  },
   "description": "Check installed versions of node and npm",
   "main": "index.js",
   "bin": {
     "check-node-version": "bin.js"
   },
+  "files": [
+    "bin.js",
+    "index.d.ts",
+    "usage.txt"
```

Comment:
One problem I have with `files` is that it’s easy to forget to add a new file to it and that bug is not caught by tests. I also personally find readmes and tests in node_modules useful when debugging/exploring.

Do you think dist package size is worth optimizing for right now? If you do, can you look at adding a test that would catch new used files not being included?

Reply:
It's not so much about size, because drives are huge, but about cleanliness and keeping packages focussed on usage, while debugging should be done in source, i.e. with the github repo. As for README, that's always included:
```sh
> npm publish --dry-run
npm notice
npm notice package: check-node-version@3.3.0
npm notice === Tarball Contents ===
npm notice 1.3kB package.json
npm notice 2.7kB bin.js
npm notice 622B  CHANGELOG.md
npm notice 2.7kB index.d.ts
npm notice 4.2kB index.js
npm notice 1.2kB LICENSE
npm notice 5.2kB README.md
npm notice 2.8kB README.src.md
npm notice 1.1kB usage.txt
```

Looking at that, I'd personally also exclude `README.src.md`.

However, it's your package, and if you want to include everything, that's what's going to happen.

Testing for used files is very much a hassle. Especially for a project of this size. However, there's always the option of blacklisting with `.npmignore`, which is my second option.

Reply:
Agreed on testing being more of a hassle than it's worth. Blacklisting would be less error prone for new files being added -- I'd be ok with that.